tests/unit/askPresentIdeas.test.ts

```diff
@@ -127,4 +135,58 @@ describe("Daily cron", () => {
 
     expect(eventBridge.send).not.toHaveBeenCalled();
   });
+
+  it("Should publish askPresentAndSquadJoinFromTeam event with a schedule", async () => {
```

Comment:
Thats why i have concers about megring test, because the name than should be something like Should publish askPresentAndSquadJoinFromTeam, createBirthdaySquad, birthdayCleanup events with a schedule. But thats looks a bit strange or dont know.

Reply:
I renamed it to Should publish scheduled birthday events if user has birthday exactly 2 months from now